lib/stomp.dart

```diff
@@ -49,60 +41,77 @@ class StompClient {
     }
 
     _handler = StompHandler(
-        config: config.copyWith(onConnect: (_, frame) {
-      if (!_isActive) {
-        config.onDebugMessage(
-            '[STOMP] Client connected while being deactivated. Will disconnect');
-        _handler?.dispose();
-        return;
-      }
-      config.onConnect(this, frame);
-    }, onWebSocketDone: () {
-      config.onWebSocketDone();
-
-      if (_isActive) {
-        _scheduleReconnect();
-      }
-    }));
-    _handler.start();
+      config: config.copyWith(
+        onConnect: (_, frame) {
```

Comment:
Not quite sure, what you propose here. Do you mean passing the separate (internal) callback to the `StompHandler` directly, like so?

```dart
StompHandler({required this.config, required this.onConnected}) { ... }
```

Another option would be to simply remove the client parameter alltogether. It seems to be that everybody needs to keep a reference to the `StompClient` around anyway. Or is there a use case I'm not seeing?

Reply:
Yeah i think you are right. We can just remove it. I don't use it in my project aswell and i did not come up with any use case where this would be helpful.

Reply:
Allright, I removed it!